clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -319,6 +322,10 @@ public void close() {
         }
         sensors.close();
         channelBuilder.close();
+        for (Map.Entry<String, KafkaChannel> entry : this.closingChannels.entrySet()) {
```

Comment:
I was trying to find the source of this logic. I don't see it in trunk or in #181.